lib/Reader/reader.cpp

```diff
@@ -6564,6 +6569,7 @@ void ReaderBase::readBytesForFlowGraphNodeHelper(
       goto GEN_BRANCH;
 
     GEN_BRANCH:
+      nop();
```

Comment:
This was another instance of needed a nop to make the debugger happy, in particular with breaks. This was caught by one of the automated debugger tests that had breaks in it. Basically this nop maps to the il offset of where the curly brace source line is.